package.json

```diff
@@ -44,8 +44,8 @@
     "@hapipal/toys": "3.x.x",
     "bin-v8-flags-filter": ">=1.2.0 <2",
     "glob": "7.x.x",
-    "marked": "1.x.x",
-    "marked-terminal": "4.x.x",
+    "marked": "4.0.16",
```

Comment:
Is there a special reason this version is pinned, or can we use `^4.0.16`?

Reply:
The markdown parsing change comes in at 4.0.17, which breaks some of hpal's tests, so I pinned it to this specific version.